src/Experimental/Form.fs

```diff
@@ -0,0 +1,256 @@
+﻿module Suave.Form
+
+open Microsoft.FSharp.Reflection
+open Microsoft.FSharp.Quotations
+open Microsoft.FSharp.Quotations.Patterns
+
+open System
+open System.Text.RegularExpressions
+
+open Suave.Html
+open Suave.Utils
+open Suave.Types
+open Suave.Model
+
+[<Literal>]
+let emailPattern = @"\A(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?)\Z"
```

Comment:
There are no valid e-mail regex patterns.

You're better off using MailAddress from System.Net.Mail.